cc-ui/src/components/form/ConfigurationForm.tsx

```diff
@@ -1,153 +1,153 @@
-                mr='0'
-                ml='0'
-              >
-                Käytä näitä Yhteyshenkilön tietoja kaikissa tämän kartoituksen
-                materiaaleissa
-              </FormLabel>
-              <Text width='10%' as='i'>
-                {values.contactInformation ? 'Kyllä' : 'Ei'}
-              </Text>
-              <Switch
-                isChecked={values.contactInformation}
-                id='contactInformation'
-                onChange={(e) => {
-                  setFieldValue('contactInformation', e.target.checked);
-                }}
-              />
-            </FormControl>
-
-            <FormControl
-              display='flex'
-              width='100%'
-              alignItems='center'
-              p='2'
-              backgroundColor='#fff'
-              border='1px'
-              borderRadius='5px'
-            >
-              <FormLabel
-                htmlFor='expiryDate'
-                width='80%'
-                mt='auto'
-                mb='auto'
-                mr='0'
-                ml='0'
-              >
-                Käytä annettua voimassaoloaikaa kaikissa tämän kartoituksen
-                materiaaleissa
-              </FormLabel>
-              <Text width='10%' as='i'>
-                {values.expiryDate ? 'Kyllä' : 'Ei'}
-              </Text>
-              <Switch
-                isChecked={values.expiryDate}
-                id='expiryDate'
-                onChange={(e) => {
-                  setFieldValue('expiryDate', e.target.checked);
-                }}
-              />
-            </FormControl>
-            <FormControl
-              display='flex'
-              width='100%'
-              alignItems='center'
-              p='2'
-              backgroundColor='#fff'
-              border='1px'
-              borderRadius='5px'
-            >
-              <FormLabel
-                htmlFor='location'
-                width='80%'
-                mt='auto'
-                mb='auto'
-                mr='0'
-                ml='0'
-              >
-                Käytä annettua sijaintia kaikissa tämän kartoituksen
-                materiaaleissa
-              </FormLabel>
-              <Text width='10%' as='i'>
-                {values.location ? 'Kyllä' : 'Ei'}
-              </Text>
-              <Switch
-                isChecked={values.location}
-                id='location'
-                onChange={(e) => {
-                  setFieldValue('location', e.target.checked);
-                }}
-              />
-            </FormControl>
-            <Flex width='100%'>
-              <Spacer />
-              <Button
-                type='submit'
-                colorScheme='blue'
-                borderRadius='0'
-                textTransform='uppercase'
-              >
-                Tallenna
-              </Button>
-            </Flex>
-          </VStack>
-        </form>
-      )}
-    </Formik>
-  );
-};
+import {

```

Comment:
what has changed here?

Reply:
There shouldn't either be anything changed